modules/lfs/content_store.go

```diff
@@ -0,0 +1,94 @@
+package lfs
+
+import (
+	"crypto/sha256"
+	"encoding/hex"
+	"errors"
+	"code.gitea.io/gitea/models"
+	"io"
+	"os"
+	"path/filepath"
+)
+
+var (
+	errHashMismatch = errors.New("Content hash does not match OID")
+	errSizeMismatch = errors.New("Content size does not match")
+)
+
+// ContentStore provides a simple file system based storage.
+type ContentStore struct {
+	basePath string
+}
+
+// Get takes a Meta object and retreives the content from the store, returning
+// it as an io.Reader. If fromByte > 0, the reader starts from that byte
+func (s *ContentStore) Get(meta *models.LFSMetaObject, fromByte int64) (io.Reader, error) {
+	path := filepath.Join(s.basePath, transformKey(meta.Oid))
+
+	f, err := os.Open(path)
```

Comment:
Turns out the issue was already filed as https://github.com/git-lfs/lfs-test-server/pull/49 with the lfs-test-server project since it makes unit tests on Windows fail.